core/mpi/controller/paciente.ts

```diff
@@ -923,3 +925,159 @@ export async function actualizarGeoReferencia(dataPaciente, req) {
     }
 }
 
+export async function getHistorialPaciente(req, dataPaciente) {
+    try {
+        let pipelineTurno = [];
+        let pipelineSobreturno = [];
+        if (req.query.turnosProximos) {
```

Comment:
Yo remplazaría este flag por dos parametros: `desde`, `hasta`. Y desde la app si queres los próximos turnos mandas `desde=now()`. 
También agregaría skip and limit, aunque sea echo por memoria, después de un tiempo los pacientes van a tender a tener muchos turnos y no esta bien que se manden todos juntos. Hay que estar precavidos. 

Y si te la querés jugar por el equipo, acá tenes info de como refactorizar la query para obtener turnos y sobreturnos en una sola query: 

https://github.com/andes/api/wiki/CITAS#listar-turnos-y-sobreturnos

Reply:
Sumados flag